docs/reference-guide/import-data/overview.md

```diff
@@ -4,11 +4,8 @@ title: Import data
 sidebar_label: Import data overview
 ---
 
-There are three main methods for importing data into Memgraph:
+There are two main methods for importing data into Memgraph:
 
-- **[CSV Import Tool](/memgraph/reference-guide/import-data/csv-import-tool)**:
-  Memgraph includes the CSV import tool which allows you to import nodes and
-  relationships from multiple CSV files.
 - **[LOAD CSV Clause](/reference-guide/import-data/load-csv-clause.md)**: You
   can also use the Cypher clause `LOAD CSV` to load data from CSV files.
```

Comment:
delete "also"

I would rewrite it to: You can use `LOAD CSV` Cypher clause to load data from CSV files.